src/scss/fotorama.scss

```diff
@@ -18,7 +18,7 @@ $blue: mix(#00d1ff, #008ed6);
   }
 }
 
-.fullscreen {
+.fotorama__fullscreen {
```

Comment:
maybe we need to name it more strict? like “fotorama opened in fullscreen” or “document in fotorama-fullscreen mode”